spec/e2e/accessibility.test.js

```diff
@@ -54,6 +54,8 @@ describe('accessibility', () => {
           'wcag22aa',
           'best-practice',
         ])
+        .exclude('div[data-touchpoints-form-id]')
+        // .exclude('a#fba-button[data-open-modal]')
```

Comment:
Having fixed the problem, I we can bring back in accessibility checks for the feedback button itself. 

The form that the button pops up may still trigger accessibility violations if the checker thinks that it obscures links underneath it, and we can visit that in a follow-up PR.